tests/lambda_endpoints/test_indexer_lambda.py

```diff
@@ -10,20 +15,130 @@ def test_batch_job_event(test_engine):
     # leaving for now to test database capabilities.
     # Will remove in upcoming PR.
     event = {
-        "Records": [
-            {
-                "detail-type": "Object Created",
-                "source": "aws.s3",
-                "s3": {
-                    "version": "0",
-                    "bucket": {"name": "sds-data-449431850278"},
-                    "object": {
-                        "key": "imap_hit_l0_sci-test_20240101_20240104_v02-01.pkts",
-                        "reason": "PutObject",
-                    },
-                },
-            }
-        ]
+        "detail-type": "Batch Job State Change",
+        "source": "aws.batch",
+        "detail": {
+            "jobName": "test-batch-tenzin",
+            "status": "FAILED",
+            "statusReason": "some error message",
+            "container": {
+                "image": (
+                    "123456789012.dkr.ecr.us-west-2.amazonaws.com/"
+                    "codicehi-repo:latest"
+                ),
+                "command": [
+                    "python",
+                    (
+                        "imap_cli --instrument glows --level l1a "
+                        "--filename '<s3-filepath>'"
+                    ),
+                ],
+            },
+        },
+    }
+    returned_value = indexer.lambda_handler(event=event, context={})
+    assert returned_value["statusCode"] == 200
+
+    event["detail-type"] = "Testing"
+    returned_value = indexer.lambda_handler(event=event, context={})
+    assert returned_value["statusCode"] == 400
+    assert returned_value["body"] == "Invalid event"
+
+
+def test_custom_lambda_event(test_engine):
```

Comment:
Could you add a quick description of what you are testing/expecting to happen in each of your assert blocks? I think that will make it easier to follow what I should be looking for.

```
# With an input event with two dependencies there should be XYZ in the database table

# Here we are making sure that a SWAPI file is in the table

# Test with a bad job status (must be INPROGRESS)
```